lib/components/UploadList.jsx

```diff
@@ -69,13 +69,19 @@ var UploadList = React.createClass({
     var showDetailsThisUpload = _.includes(this.state.showErrorDetails, upload.key);
     var errorDetails = showDetailsThisUpload ? (<div className="UploadList-error-details">{upload.error.debug}</div>) : null;
     var showErrorsText = showDetailsThisUpload ? this.props.text.HIDE_ERROR : this.props.text.SHOW_ERROR;
-    
+    var errorMessage = upload.error.driverLink ? <div className="UploadList-error-message-wrapper">
```

Comment:
Mmkay, just tried w/no Dexcom attached to see the error. Is the detail message re: powering off and on what's expected? I think it is, but wanted to check w/you to be certain:

<img width="487" alt="screenshot 2016-01-14 10 28 57" src="https://cloud.githubusercontent.com/assets/1588547/12333562/bc7ec716-baa9-11e5-821a-50c8d2ef7d69.png">